pennylane/ops/qubit/attributes.py

```diff
@@ -158,3 +165,23 @@ def __contains__(self, obj):
 ``qml.Toffoli(wires=[1, 0, 2])``, but neither are the same as
 ``qml.Toffoli(wires=[0, 2, 1])``.
 """
+
+diagonal_in_z_basis = Attribute(
+    [
+        "PauliZ",
+        "S",
+        "T",
+        "CZ",
+        "DiagonalQubitUnitary",
+        "RZ",
+        "PhaseShift",
+        "ControlledPhaseShift",
```

Comment:
Just can't figure out where this whitespace is supposed to be!

Reply:
That's very odd - I pulled the branch to take a look in my editor, and I don't see it either :confused:

Reply:
huh, super weird. Black should be removing all trailing whitespaces.

Maybe codefactor is just confused